.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
- [ ] Kindly make sure you only have useful files in your repository. This `.idea` doesn't have any meaning being here. Kindly consider removing that.